style.css

```diff
@@ -2,6 +2,56 @@
   padding-inline-start: 0;
 }
 
+.overlay {
+  height: 0;
+  width: 100%;
+  position: fixed;
+  z-index: 1;
+  top: 0;
+  left: 0;
+  background-color: rgba(96, 112, 255, 1);
+  overflow-x: hidden;
+  transition: 0.5s;
+  mix-blend-mode: multiply;
```

Comment:
- [ ] Use also `backdrop-filter: blur(8px);` for the blur effect as the original design